R/Steph_Function_Attempts.R

```diff
@@ -0,0 +1,101 @@
+##This is my script for writing a function to edit an attribute in an existing attributes table.
+
+library(testthat)
+library(arcticdatautils)
+library(dataone)
+library(EML)
+library(devtools)
+
+cnTest <- dataone::CNode('STAGING')
+mnTest <- dataone::getMNode(cnTest,'urn:node:mnTestARCTIC')
+
+#dummy package
+pkg <- arcticdatautils::create_dummy_package(mnTest,
+                                             size = 2)
+#dummy data table:
+attributes1 <- data.frame(
```

Comment:
Ultimately we don't want this test code to be in the same file as the function, but totally fine for now.